### PR TITLE
Heretics mark: Abruptly cancel opponent's move for specific edge cases

### PR DESCRIPTION
**Pull request by ghost**
not tested yet, but in theory should work

### CODE REVIEW

**Comment by ghost** [comment]
now its tested

**Comment by HoeenCoder** [comment]
@scheibo can you take a look at this when you have time? I want multiple people to look for potential issues on this one. (I'll be digging myself soon).

File: data/mods/ssb/moves.js [hunk 1]
```diff
@@ -495,6 +495,7 @@ let BattleMovedex = {
 			this.add('-anim', target, 'Dark Void', target);
 		},
 		onHit(target, source, move) {
+			let wouldMove = this.cancelMove(target);
```
Comment (scheibo) [inline data/mods/ssb/moves.js:498]: Looking through the codebase for where else we use cancelMove this LGTM

**Review by scheibo — APPROVED** [review]

**Review by HoeenCoder — APPROVED** [review]
Looks good now

**Pull request merged**